examples/evaluate/06-TBROM_input_field.py

```diff
@@ -0,0 +1,195 @@
+""".. _ref_example_TBROM_images:
+
+3D field ROM example for input field snapshot projection as well as snapshot generation on demand
+-------------------------------------------------------------------------------------------------
+
+This example shows how PyTwin can be used to load and evaluate a twin model which has a field ROM with inputs
+parameterized by both scalar and field data. The example shows also how to evaluate the output field data in the form
+of snapshots.
+
+.. note::
+   To be able to use the functionalities to project an input field snapshot, you need to have a Twin with 1 or more
+   TBROM parameterized by input field data, for which its input mode coefficients are connected to Twin inputs with the
+   following convention :
+   if there are multiple TBROM in the Twin, Twin input named as : "input field name"_mode_"i"_"tbrom name"
+   if there is a single TBROM in the Twin, Twin input named as : "input field name"_mode_"i"
+
+   To be able to use the functionalities to generate output field snapshot on demand, you need to have a Twin with 1 or
+   more TBROM, for which its output mode coefficients are enabled when exporting the TBROM, and connected to Twin
+   outputs with the following convention :
+   if there are multiple TBROM in the Twin, Twin output named as : outField_mode_"i"_"tbrom name"
+   if there is a single TBROM in the Twin, Twin output named as : outField_mode_"i"
+
+   To be able to use the functionalities to generate points file on demand, you need to have a Twin with 1 or
+   more TBROM, for which its geometry is embedded when exporting the ROM to Twin Builder
```

Comment:
```suggestion
   To be able to use the functionalities to generate a points file on demand, you must
    have a twin with one or more TBROMs. Geometries for the TBROMs must be
    embedded when exporting the ROM to Twin Builder.
```
Should the last sentence be "... exporting the ROMs to Twin Builder." Or, ...exporting the TROMs to Twin Builder? 
Once you say there can be one or more , subsequent information can use the plural to assume that there are more than one.

Reply:
modified to "exporting the TBROMs..."